test/integration/licence_test.rb

```diff
@@ -746,4 +746,279 @@ class LicenceTest < ActionDispatch::IntegrationTest
+                  "introduction" => "",
+                  "usesLicensify" => "true"
+                }
+              ]
+            }
+          }
+        ]
+
+        licence_exists('1071-5-1',
+                       "isLocationSpecific" => false,
+                       "geographicalAvailability" => %w(England Wales),
+                       "issuingAuthorities" => authorities)
+
+        visit '/licence-to-kill'
+      end
+
+      should "display the title and authority" do
+        assert page.has_content? 'Licence to kill'
+        assert page.has_content? 'Ministry of Love'
+      end
+     
+      should "not show licence actions that don't have the usesLicensify param" do
+        within("#content nav") do
+          refute page.has_link? "How to apply", href: '/licence-to-kill/miniluv/apply'
+        end
+      end
+
+      should "show licence actions that have usesLicensify set to true" do
+        within("#content nav") do
+          assert page.has_link? "How to renew", href: '/licence-to-kill/miniluv/renew'
+        end
+      end
+  
+      should "display interactions for licence" do
+        click_on "How to renew"
+        assert current_path == '/licence-to-kill/miniluv/renew'
+        assert page.has_link? "Apply online", href: '/licence-to-kill/ministry-of-love/renew-1'
+      end
+    end
+
+    context "when usesLicensify is missing for all actions" do
+      setup do
+        authorities = [
+          {
+            "authorityName" => "Ministry of Love",
+            "authoritySlug" => "miniluv",
+            "authorityInteractions" => {
+              "apply" => [
+                {
+                  "url" => "/licence-to-kill/ministry-of-love/apply-1",
+                  "description" => "Apply for your licence",
+                  "payment" => "none",
+                  "introduction" => ""
+                },
+              ],
+              "renew" => [
+                {
+                  "url" => "/licence-to-kill/ministry-of-love/renew-1",
+                  "description" => "Apply for your licence",
+                  "payment" => "none",
+                  "introduction" => "",
+                }
+              ]
+            }
+          }
+        ]
+    
+        licence_exists('1071-5-1',
+                       "isLocationSpecific" => false,
+                       "geographicalAvailability" => %w(England Wales),
+                       "issuingAuthorities" => authorities)
+    
+        visit '/licence-to-kill'
+      end
+
+      should "display the title" do
+        assert page.has_content?('Licence to kill')
+      end
+
+      should "not display authority" do
+        refute page.has_content? 'Ministry of Love'
+        refute page.has_button? 'Get started'
+      end
+
+      should "display the licence unavailable message" do
+        assert page.has_content?("You can't apply for this licence online")
+        assert page.has_content?("Contact your local council")
+      end
+    end
```

Comment:
These integration tests are super useful.  What we don't have is a test covering the (possibly apocryphal?) scenario where a single action has multiple links, and what we do in those cases (currently hide the ones without `usesLicensify` or with `usesLicensify` set to `false`).

Reply:
Added tests for multiple link scenario.